src/layer/layer_settings_manager.cpp

```diff
@@ -50,23 +50,25 @@ static std::string GetAndroidProperty(const char *name) {
 }
 #endif
 
-static bool IsEnvironment(const char *variable) {
-#if defined(__ANDROID__)
-    return !GetAndroidProperty(variable).empty();
-#else
-    return std::getenv(variable) != NULL;
-#endif
-}
-
 static std::string GetEnvironment(const char *variable) {
 #if defined(__ANDROID__)
-    return GetAndroidProperty(variable);
+    std::string result = GetAndroidProperty(variable);
+    // Workaround for screenshot layer backward compatibility
+    if (result.empty() && std::string(variable) == "debug.vulkan.screenshot.frames") {
+        result = GetAndroidProperty("debug.vulkan.screenshot");
+    }
+    return result;
```

Comment:
Kinda awkward to have this workaround in this repository just for the screen shot repo.